contracts/bridge/BridgeTransferERC20.sol

```diff
@@ -0,0 +1,124 @@
+pragma solidity ^0.4.24;
+
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/ERC20Burnable.sol";
+
+import "../sc_erc20/IERC20BridgeReceiver.sol";
+import "./BridgeTransferCommon.sol";
+
+contract BridgeTransferERC20 is IERC20BridgeReceiver, BridgeTransferCommon{
+    // handleERC20Transfer sends the token by the request.
+    function handleERC20Transfer(
+        bytes32 _requestTxHash,
+        address _from,
+        address _to,
+        address _tokenAddress,
+        uint256 _value,
+        uint64 _requestedNonce,
+        uint64 _requestedBlockNumber,
+        uint256[] _extraData
+    )
+        public
+        onlyOperators
+    {
+        if (!voteValueTransfer(_requestedNonce)) {
+            return;
+        }
+
+        emit HandleValueTransfer(
+            _requestTxHash,
+            TokenType.ERC20,
+            _from,
+            _to,
+            _tokenAddress,
+            _value,
+            _requestedNonce,
+            _extraData
+        );
+        _setHandledRequestTxHash(_requestTxHash);
+        lastHandledRequestBlockNumber = _requestedBlockNumber;
+
+        updateHandleNonce(_requestedNonce);
+
+        if (modeMintBurn) {
+            ERC20Mintable(_tokenAddress).mint(_to, _value);
+        } else {
+            IERC20(_tokenAddress).transfer(_to, _value);
+        }
+    }
+
+    // _requestERC20Transfer requests transfer ERC20 to _to on relative chain.
+    function _requestERC20Transfer(
+        address _tokenAddress,
+        address _from,
+        address _to,
+        uint256 _value,
+        uint256 _feeLimit,
+        uint256[] _extraData
+    )
+        internal
+    {
+        require(isRunning, "stopped bridge");
+        require(_value > 0, "zero msg.value");
+        require(allowedTokens[_tokenAddress] != address(0), "invalid token");
+
+        uint256 fee = _payERC20FeeAndRefundChange(_from, _tokenAddress, _feeLimit);
+
+        if (modeMintBurn) {
+            ERC20Burnable(_tokenAddress).burn(_value);
+        }
+
+        emit RequestValueTransfer(
+            TokenType.ERC20,
+            _from,
+            _to,
+            _tokenAddress,
+            _value,
+            requestNonce,
+            "",
+            fee,
+            _extraData
+        );
+        requestNonce++;
+    }
+
+    // Receiver function of ERC20 token for 1-step deposits to the Bridge
+    function onERC20Received(
+        address _from,
+        uint256 _value,
+        address _to,
```

Comment:
@KimKyungup , Could you change the order of `_value` and `_to` to match up to the order of `_requestERC20Transfer`?

Reply:
OK.  It's good idea. It make a change in golang code. I will do in next PR. :)

Reply:
I thought wrong. I can fix it without any modification of core code.

Reply:
reflected  :) Thanks.

Reply:
Thank you!